lightwood/data/statistical_analysis.py

```diff
@@ -84,7 +106,7 @@ def statistical_analysis(data: pd.DataFrame,
     for col in df.columns:
         histograms[col] = None
         buckets[col] = None
-        if dtypes[col] in (dtype.categorical, dtype.binary, dtype.date):
+        if dtypes[col] in (dtype.categorical, dtype.binary, dtype.tags):
```

Comment:
Maybe split `tag` values and turn them into a set? That way you cover the case where a tag column does not have ordered tags.

Reply:
Where is the split necessary? This is specifically checking the type of the column.

Reply:
In `hist.keys()`. It would require using something other than `value_counts` though. 

Example: a movie dataset with a `genres` tag column. We'd like both `action,thriller` and `thriller,action` to count for the same bin, I'd presume.